src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.spec.ts

```diff
@@ -491,6 +467,17 @@ describe('LocalGameWrapperComponent', () => {
             expect(testUtils.getGameComponent().getTurn()).toBe(0);
             expect(testUtils.getGameComponent().updateBoard).toHaveBeenCalledTimes(1);
         }));
+        it('should not allow to take back when no human move has been made', fakeAsync(async() => {
```

Comment:
ça peut vouloir dire deux choses : soit pas de mouvement a été fait, soit un mouvement a été fait pas par un humain. On a l'autre ? Et peut-être clarifier le it du coup

Reply:
Vien bu ! J'ai enlevé du doubon (et du doublon plutôt pourri) et rajouté se case là !
Résumé:
 - should take back one turn when human move has been made
 - should take back two turn when playing against IA
 - should not allow to take back when only AI move has been made
 - should not allow to take back when no move has been made
 - should cancelMoveAttempt when taking back
 T'en vois d'autres ?

Reply:
should not allow to take back when AI vs. AI ?